web/internal/web_test.sh.template

```diff
@@ -38,25 +38,27 @@ fi
 
 printenv
 
-args=()
-portatnext=0
-debuggerport=0
-
-for var in "$@"; do
-  if (( "$portatnext" )); then
-    debuggerport="$var"
-    portatnext=0
-  elif [[ "$var" == --debugger_port=* ]]; then
-    debuggerport="${var#--debugger_port=}"
-  elif [[ "$var" == "--debugger_port" ]];  then
-    portatnext=1
+# Split arguments into WTL_ARGS and TEST_ARGS.
+TEST_ARGS=()
```

Comment:
Generally, variable names are ALL_CAPS if they are constants or if they are intentionally exported to the environment.

It probably doesn't matter that much. But I have a slight preference for lowercase names, as I think they're easier to read, and it makes it more clear that the variable is defined in this file (i.e., it can't be from the environment, because env vars are conventionally all caps).